### DISCUSSION TITLE
AI重現大師遺作 《哲學》死而復生

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#268] [閒聊八卦]
利用科技來修復畫作感覺已是相當常見的事了
但修復後的價值和認同可能就見仁見智了
[https://tw.news.yahoo.com/ai%E9%87%8D%E7%8F%BE%E5%A4%A7%E5%B8%AB%E9%81%BA%E4%BD%9C-%E5%93%B2%E5%AD%B8-%E6%AD%BB%E8%80%8C%E5%BE%A9%E7%94%9F-075645044.html](url)

**Comment by zixunn** [comment]
雖然歷史名畫家珍貴的作品在時間洪流中遭受到損壞是非常可惜的事，然而歷史痕跡本身也是藝術的一部分，用AI來修復有過於專斷與死板的感覺。

**Comment by MAKERRRofKEVIN** [comment]
修復後的作品也未嘗不是一件新作品，若我們不一昧比較復原的完全度，從欣賞藝術與科技結合的角度也能看到新興藝術之美

**Comment by syt-110** [comment]
覺得畫作的部分還是需要相關專業人士修復，因為名畫通常考慮到筆觸部份以及原始作品本身並不是完美的，通常整體呈現的感覺需要看繪師當下情緒和想表達之感受而定，因此我覺得文藝作品不太能用AI修復。
